hr_employee_attendance_report/tests/test_hr_employee_attendance_report.py

```diff
@@ -0,0 +1,103 @@
+                "company_id": cls.company.id,
+            }
+        )
+
+        cls.leave_type = cls.env["hr.leave.type"].create(
+            {
+                "name": "Time Off",
+                "time_type": "leave",
+                "requires_allocation": "yes",
+                "allocation_validation_type": "no",
+            }
+        )
+        cls.leave = cls.env["hr.leave"].create(
+            {
+                "name": "Leave test",
+                "holiday_status_id": cls.leave_type.id,
+                "date_from": datetime(2021, 1, 2, 1, 0, 0),
+                "date_to": datetime(2021, 1, 5, 23, 0, 0),
+                "request_date_from": datetime(2021, 1, 2, 8, 0, 0),
+                "request_date_to": datetime(2021, 1, 5, 17, 0, 0),
+                "number_of_days": 1,
+                "employee_id": cls.employee.id,
+            }
+        )
+        cls.allocation = cls.env["hr.leave.allocation"].create(
+            {
+                "name": "Some Holiday",
+                "holiday_type": "employee",
+                "employee_ids": [(4, cls.employee.id)],
+                "employee_id": cls.employee.id,
+                "holiday_status_id": cls.leave_type.id,
+                "number_of_days": 2,
+                "allocation_type": "regular",
+                "date_from": datetime(2021, 1, 2, 8),
+                "date_to": datetime(2021, 1, 3, 17),
+            }
+        )
+        cls.resource_leave = cls.env["resource.calendar.leaves"].create(
+            [
+                {
+                    "name": "Absence",
+                    "company_id": cls.company.id,
+                    "calendar_id": cls.employee.resource_calendar_id.id,
+                    "resource_id": cls.employee.resource_id.id,
+                    "holiday_id": cls.leave.id,
+                    "date_from": datetime(2021, 1, 2, 8, 0, 0),
+                    "date_to": datetime(2021, 1, 2, 17, 0, 0),
+                    "time_type": "leave",
+                }
+            ]
+        )
+
+        cls.env["hr.attendance"].create(
+            {
+                "employee_id": cls.employee.id,
+                "check_in": datetime(2021, 1, 4, 8, 0),
+                "check_out": datetime(2021, 1, 4, 20, 0),
+            }
+        )
+        cls.env["hr.attendance.overtime"].create(
+            {"employee_id": cls.employee.id, "duration": 3.0}
+        )
+
+    def test_report_hr_attendance_value(self):
+        data = {"date_from": "2021-01-02", "date_until": "2021-01-04"}
+        values = self.report._get_report_values(docids=self.employee.id, data=data)
+        summary = values["summary"].get(self.employee.id)
+        leave_allocations = values["leave_allocations"].get(self.employee.id)
+        self.assertEqual(summary.get("worked_hours"), 11.0)
+        self.assertEqual(summary.get("overtime_total"), 6.0)
+        self.assertEqual(summary.get("leave_hours"), 16.0)
+        self.assertEqual(summary.get("overtime"), 3.0)
```

Comment:
why overtime is 3.0 but overtime_total is 6.0?

Reply:
Overtimes that not created from [employee attendance (check in/out)](https://github.com/sonhd91/hr-attendance/blob/17.0-hr_employee_attendance_report/hr_employee_attendance_report/tests/test_hr_employee_attendance_report.py#L87) will be calculated as overtimes, since it's has date set init while overtime that [registered ](https://github.com/sonhd91/hr-attendance/blob/17.0-hr_employee_attendance_report/hr_employee_attendance_report/tests/test_hr_employee_attendance_report.py#L91)will not be calculated